api/src/services/project-service.ts

```diff
@@ -342,7 +343,13 @@ export class ProjectService extends DBService {
     await this.insertParticipantRole(projectId, PROJECT_ROLE.PROJECT_LEAD);
 
     //Submit Eml to biohub
-    await this.platformService.submitDwCAMetadataPackage(projectId);
+    const queueResponse = await this.platformService.submitDwCAMetadataPackage(projectId);
+
+    // take queue id and insert into history publish table
+    if (queueResponse?.queue_id) {
```

Comment:
Do we need this check? The return type of `platformService.submitDwCAMetadataPackage` implies that this branch will always be taken.

Reply:
the `submitDwCAMetaDataPackage` function can possibly fail (try catch) but currently isn't throwing so an undefined is a valid response. Good point of discussion though, should `submitDwCAMetaDataPackage` silently fail? should it just throw the error up at let the parent handle it? @NickPhura @KjartanE

Reply:
I would move this whole check and history table call into the platform service, and then squash any errors there. I don't think we want to prevent a project from being created just because the submit fails. And now that we have the history tables, we can maybe build out something to inform the user or admins, later.

Reply:
Could probably even not await this call here, since it doesn't impact the surrounding code.

Reply:
Unless it relies on the current db connection, then we would need to await, so the connection doesn't get closed in the endpoint before its done running.

Reply:
added a new function call to wrap the submit and publish inserts